frame/election-provider-support/src/lib.rs

```diff
@@ -618,3 +628,99 @@ pub type Voter<AccountId, Bound> = (AccountId, VoteWeight, BoundedVec<AccountId,
 /// Same as [`Voter`], but parameterized by an [`ElectionDataProvider`].
 pub type VoterOf<D> =
 	Voter<<D as ElectionDataProvider>::AccountId, <D as ElectionDataProvider>::MaxVotesPerVoter>;
+
+/// A bounded equivalent to [`sp_npos_elections::Support`].
+#[derive(Default, RuntimeDebug, Encode, Decode, scale_info::TypeInfo, MaxEncodedLen)]
+#[codec(mel_bound(AccountId: MaxEncodedLen, Bound: Get<u32>))]
+#[scale_info(skip_type_params(Bound))]
+pub struct BoundedSupport<AccountId, Bound: Get<u32>> {
+	/// Total support.
+	pub total: ExtendedBalance,
+	/// Support from voters.
+	pub voters: BoundedVec<(AccountId, ExtendedBalance), Bound>,
+}
+
+pub type BoundedSupportOf<E> = BoundedSupport<
+	<E as ElectionProvider>::AccountId,
+	<E as ElectionProvider>::MaxBackersPerWinner,
+>;
+
+pub type BoundedSupports<AccountId, MaxBackersPerWinner> =
+	Vec<(AccountId, BoundedSupport<AccountId, MaxBackersPerWinner>)>;
+
+/// Convenience to create a [`BoundedSupport`] from a [`ElectionProvider`].
```

Comment:
```suggestion
/// Convenience to create a [`BoundedSupports`] from a [`ElectionProvider`].
```

I am very open to the idea of not relying on this damn 's' to distinguish Support and Supports. Suggestions?